server/views/pages/bookAVisit/mainContact.njk

```diff
@@ -12,83 +12,118 @@
+              checked: visitor.personId == formValues.contact
+            }
+          ), visitors) %}
+        {% endfor %}
 
-            {% set someoneElseHtml %}
-                {{ govukInput({
-                    id: "someoneElseName",
-                    name: "someoneElseName",
-                    type: "text",
-                    classes: "govuk-!-width-one-third",
-                    label: {
-                        text: "Name"
-                    },
-                    value: formValues.someoneElseName,
-                    autocomplete: "off",
-                    errorMessage: errors | findError('someoneElseName')
-                }) }}
-            {% endset -%}
-            
-            {% set visitors = (visitors.push(
-                {
-                    value: "someoneElse",
-                    text: "Someone else",
-                    checked: formValues.contact == "someoneElse",
-                    conditional: {
-                        html: someoneElseHtml
-                    }
-                }
-            ), visitors) %}
+        {% set someoneElseHtml %}
+          {{ govukInput({
+            id: "someoneElseName",
+            name: "someoneElseName",
+            type: "text",
+            classes: "govuk-!-width-one-third",
+            label: {
+                text: "Name"
+            },
+            value: formValues.someoneElseName,
+            autocomplete: "off",
+            errorMessage: errors | findError('someoneElseName')
+          }) }}
+        {% endset -%}
+          
+        {% set visitors = (visitors.push(
+          {
+            value: "someoneElse",
+            text: "Someone else",
+            checked: formValues.contact == "someoneElse",
+            conditional: {
+              html: someoneElseHtml
+            }
+          }
+        ), visitors) %}
 
-            {{ govukRadios({
-                idPrefix: "contact",
-                name: "contact",
-                fieldset: {
-                    legend: {
-                        text: pageHeaderTitle,
-                        isPageHeading: true,
-                        classes: "govuk-fieldset__legend--l"
-                    }
-                },
-                items: visitors,
-                errorMessage: errors | findError('contact')
-            }) }}
+        {{ govukRadios({
+          idPrefix: "contact",
+          name: "contact",
+          fieldset: {
+            legend: {
+              text: pageHeaderTitle,
+              isPageHeading: true,
+              classes: "govuk-fieldset__legend--l"
+            }
+          },
+          items: visitors,
+          errorMessage: errors | findError('contact')
+        }) }}
+        
+        {% set inputPhoneNumberHtml %}
+          {{ govukInput({
+            classes: "govuk-!-width-one-third",
+            label: {
+              text: "Number"
+            },
+            id: "phoneNumberInput",
+            name: "phoneNumberInput",
```

Comment:
Doesn't really matter, but `id / name` could have remained `phoneNumber` so fewer changes rippling through 🤷‍♂️. I think we've ended up with a field called `phoneNumber` which **isn't** the phone number. 

Perhaps would be clearer if the radios input were named `hasPhoneNumber` with values of `yes | no` and the phone number text input was `phoneNumber`. Lines 19-20 & 53 in `mainContact.ts` might then be easier to understand.